### PR TITLE
PVS/V618: fix printf-style args

### PR DESCRIPTION
**Pull request by justinmk**
API errors are not translated. https://github.com/neovim/neovim/issues/6150

### CODE REVIEW

File: src/nvim/api/buffer.c [hunk 1]
```diff
@@ -1123,7 +1123,7 @@ ArrayOf(Integer) nvim_buf_get_extmark_by_id(Buffer buffer, Integer ns_id,
   }
 
   if (!ns_initialized((uint64_t)ns_id)) {
-    api_set_error(err, kErrorTypeValidation, _("Invalid ns_id"));
+    api_set_error(err, kErrorTypeValidation, "Invalid ns_id");
```
Comment (justinmk) [inline src/nvim/api/buffer.c]: We don't translated API errors. https://github.com/neovim/neovim/issues/6150

**Comment by justinmk** [comment]
@janlazo any idea why travis macos build is complaining about missing `cpanm` lately?

    sudo: cpanm: command not found

**Comment by janlazo** [comment]
```
6.70s$ rvm $brew_ruby do brew bundle --verbose --global
/usr/local/bin/brew tap homebrew/bundle
==> Tapping homebrew/bundle
Cloning into '/usr/local/Homebrew/Library/Taps/homebrew/homebrew-bundle'...
Tapped (102 files, 251.1KB).
Error: Unknown command: bundle
```
`cpanminus` wasn't installed?

**Comment by justinmk** [comment]
Possibly homebrew made a breaking change.

**Comment by justinmk** [comment]
Thanks @janlazo , I pushed `update: true` workaround: https://github.com/neovim/neovim/commit/38cda18626fc86209068d8282ac41edea6967b88

**Pull request merged**